Gems/PhysX/Code/Editor/ColliderCapsuleMode.cpp

```diff
@@ -137,19 +170,26 @@ namespace PhysX
         {
             OnHeightManipulatorMoved(action, idPair);
         });
-
     }
 
     void ColliderCapsuleMode::OnRadiusManipulatorMoved(const AzToolsFramework::LinearManipulator::Action& action, const AZ::EntityComponentIdPair& idPair)
     {
+        // manipulator action offsets do not take entity transform scale into account, so need to apply it here
+        float transformScale = 1.0f;
+        const AZ::Transform colliderLocalTransform = Utils::GetColliderLocalTransform(idPair);
+        AZ::TransformBus::EventResult(transformScale, idPair.GetEntityId(), &AZ::TransformBus::Events::GetWorldUniformScale);
+        transformScale = AZ::GetMax(AZ::MinTransformScale, transformScale);
+        const AZ::Vector3 localPosition = colliderLocalTransform.GetInverse().TransformPoint(
+            action.m_start.m_localPosition + action.m_current.m_localPositionOffset / transformScale);
+
```

Comment:
nit: This blocks appears a couple of times. Is it worth moving it into a function?

Reply:
Yeah, that's a good call. Now that I've added the tests I'll do some refactoring. As hultonha mentioned in his comments the ideal eventual solution should involve some refactoring in the manipulators themselves so the logic here can be less complex. Lifting it out into a function like you suggest will make it easier to update that when that change is made to the manipulators.